info.yml

```diff
@@ -126,6 +126,11 @@ configuration:
         default_value: True
         default_value_tk-hiero: False
 
+    auto_expand_tree:
+        type: bool
+        description: Automatically expands the root level of the treeviews.
+        default_value: True
```

Comment:
Hey all, sorry missed this until Phil's recent update.

I agree we should set the default value to False.  We weren't even going to offer this as an option in the ticket I remember for this.  :)